oeps/oep-0001.rst

```diff
@@ -45,163 +61,211 @@ OEP Types
-sure the idea is applicable to the entire community and not just the author.
-Just because an idea sounds good to the author does not mean it will work for
-most Open edX installations.
-
-Initial Submission
-------------------
-
-Once the Author has asked the Open edX community whether an idea has any chance
+the potential authors time. Asking the Open edX community first if an idea is 
+original and appropriate helps prevent wasted effort. It also helps to make
+sure the idea is applicable to the entire community and not just the authors.
+
+.. _open-edx-proposals Slack channel: https://openedx.slack.com/messages/C1L370YTZ/details/
+
+Step 3. Create PR for "Draft" OEP
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once the Authors have asked the Open edX community whether an idea has any chance
 of acceptance, a draft OEP should be submitted as a pull request against the
-`central OEP repository`_. To identify the draft proposal, the Author should
+`central OEP repository`_. To identify the draft proposal, the Authors should
 check the numbered list of previous OEP pull requests and select the next 
-available number. The pull request title must start with "OEP-XXX", which 
-claims that OEP number for the included proposal.
+available number. The pull request title should be of the form 
+"OEP-XXXX-YYYY-ZZZZ", where
+
+* *XXXX* is the OEP number claimed for the included proposal
+* *YYYY* is the abbreviated type of the OEP (i.e., "proc", "bp" or "arch")
+* *ZZZZ* is a hyphenated brief (< 5 words) title of the proposal
 
 .. _central OEP repository: http://github.com/edx/open-edx-proposals
 
-After the Author drafts an OEP in a format in which they are comfortable, they
-will request an Arbiter from the edX Chief Architect. This Arbiter will be
-recorded in the "Arbiter" header on the OEP. The rest of Open edX community 
+Step 4. Request an Arbiter
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After the Authors draft an OEP in a format in which they are comfortable, they
+will request an Arbiter from the `edX Architecture Team`_ by emailing 
+`arch-team@edx.org`_. This Arbiter will be
+recorded in the "Arbiter" header on the OEP. The rest of the Open edX community 
 will be given the opportunity to comment on the OEP, with the Arbiter serving
 to keep the discussion on track and to evaluate when it has reached a final
 conclusion.
 
+.. _`arch-team@edx.org`: mailto:arch-team@edx.org
+
+Step 5. Review with Arbiter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 For an OEP to be accepted by the Arbiter, it must meet certain minimum
 criteria. It must be a clear and complete description of the proposed
-enhancement. The enhancement must represent a net improvement. The proposed
-implementation, if applicable, must pass the existing
-`Open edX Contribution Guidelines`_.
-
-.. _Open edX Contribution Guidelines: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/index.html
+enhancement. The enhancement must represent a net improvement.
```

Comment:
"Enhancement" feels like a confusing term when it comes to best practices, especially when helping to codify existing, but partially followed, best practices.